src/screens/DiscoveryScreenV2/index.js

```diff
@@ -354,13 +354,13 @@ const DiscoveryScreenV2 = ({route}) => {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     flex: 1,
     paddingTop: 60
   },
   fragmentContainer: {
     height: '100%',
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `backgroundColor` property in both `container` and `fragmentContainer` styles from `COLORS.white` to `COLORS.almostBlack` will significantly alter the appearance of these components. Ensure that this color change aligns with the overall design scheme and is accessible to all users, including those with visual impairments.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->